addons/delivery/wizard/choose_delivery_package.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class ChooseDeliveryPackage(models.TransientModel):
+    _name = 'choose.delivery.package'
+    _description = 'Choose delivery package type when there is a carrier on the picking'
+
+    stock_quant_package_id = fields.Many2one(
+        'stock.quant.package',
+        default=lambda self: self._default_stock_quant_package_id()
+    ) 
+    delivery_packaging_id = fields.Many2one(
+        'product.packaging',
+        domain=lambda self: self._domain_delivery_packaging_id(),
```

Comment:
should be passed by reference (without the `()`) or it's only evaluated at server startup I think (I had problems with that in the pas)